games-emulation/virtualjaguar/virtualjaguar-2.1.2-r1.ebuild

```diff
@@ -0,0 +1,58 @@
+# Copyright 1999-2014 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Header: /var/cvsroot/gentoo-x86/games-emulation/virtualjaguar/virtualjaguar-2.1.2.ebuild,v 1.1 2014/10/15 03:50:52 mr_bones_ Exp $
+
+EAPI=5
+inherit eutils qt4-r2 games
+
+DESCRIPTION="an Atari Jaguar emulator"
+HOMEPAGE="http://www.icculus.org/virtualjaguar/"
+SRC_URI="http://www.icculus.org/virtualjaguar/tarballs/${P}.tar.bz2"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~ppc ~x86"
+IUSE=""
+
+RDEPEND="media-libs/libsdl[joystick,opengl,sound,video]
+	sys-libs/zlib
+	virtual/opengl
+	dev-qt/qtcore:4
+	dev-qt/qtgui:4[-egl]
+	dev-qt/qtopengl:4[-egl]
+	dev-libs/libcdio"
+DEPEND="${RDEPEND}
+	>=sys-devel/gcc-4.4"
+
+S=${WORKDIR}/${PN}
+
+src_prepare() {
+	eqmake4 virtualjaguar.pro -o makefile-qt
```

Comment:
Wait a minute… why is this done in src_prepare()? It definitely belongs in src_configure()! That would explain why it was called twice :).